### DISCUSSION TITLE
Update has killed my sert up

### DISCUSSION THREAD

**Discussion by kosh66** [TomSchimansky/CustomTkinter#2047] [General]
Updated to 5.2.1 and get following error

  File "C:\Users\XXX\venv\Lib\site-packages\customtkinter\windows\ctk_tk.py", line 7, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'

**Comment by kosh66** [comment]
Solved myself

**Reply by ghost** [reply]
How did you solve it?

**Reply by Akascape** [reply]
@Goodwin6322 just install the packaging module: `pip install packaging`
